src/main/resources/application-local.yml

```diff
@@ -31,3 +41,16 @@ logging:
 spring:
   flyway:
     locations: classpath:db/migration,classpath:db/local
+  security:
```

Comment:
this config should be split into bits that are common across all environments, and bits that are specific to local/dev etc 

so i would pop ```
security:
  oauth2:
      client:
        provider:
          hmppsauth:
            token-uri: ${hmppsauth.baseurl}/oauth/token
        registration:
          communityapi:
            provider: hmppsauth
            authorization-grant-type: client_credentials
            scope: read
            ```
            in the top-level application.yml file,
            
            and ```
security: 
  oauth2:
      client:
        registration:
          communityapi:
            client-id: interventions
            client-secret: clientsecret
            ```
            in this file.